yml/cdb_latiss.yaml

```diff
@@ -914,6 +914,11 @@ tables:
     datatype: int
     description: Count of high signal-to-noise-ratio sources.
     ivoa:unit: count
+  - name: postisr_pixel_median
+    "@id": "#visit1_quicklook.postisr_pixel_median"
+    datatype: float
+    description: Median postISR pixel value.
```

Comment:
Is `postISR` sufficiently meaningful to end users?

Reply:
I'm happy to have suggestions here. I guess it depends exactly on how we end up configuring isrTask...